compilers/concrete-compiler/compiler/include/concretelang/Support/V0Parameters.h

```diff
@@ -110,6 +114,13 @@ struct CompositionRule {
 const std::vector<CompositionRule> DEFAULT_COMPOSITION_RULES = {};
 const bool DEFAULT_COMPOSABLE = false;
 
+struct ParameterRestrictions {
+  uint64_t log2_polynomial_size_min;
+  uint64_t log2_polynomial_size_max;
+  uint64_t glwe_dimension_min;
+  uint64_t glwe_dimension_max;
+};
+
```

Comment:
Do we need a separate struct here ? Can't `concrete_optimizer::ParameterRestrictions` be used directly ?

Reply:
Also, I would use optionals here.